src/i18n/locales/en.json

```diff
@@ -1,5 +1,94 @@
 {
   "welcome": {
-    "heading": "The power of lightning in your browser"
+    "heading": "The power of lightning in your browser",
+    "nav": {
+      "welcome": "Welcome",
+      "password": "Your Password",
+      "connect": "Connect to Lightning",
+      "done": "Done"
+    },
+    "intro": {
+      "send": "Send in One Click",
+      "send_description": "Lightning transactions happen all in your browser. No alt+tab or QR-code scanning needed.",
+      "paywall": "No more annoying paywalls",
+      "paywall_description": "Define individual budgets for websites to enable seamless payment streams. No more annoying paywalls.",
+      "privacy": "Privacy first",
+      "privacy_description": "Use lightning to authenticate and control your privacy.",
+      "foss": "Free and Open Source",
+      "foss_description": "Completely open code that can be audited by anyone. No stats or trackers. You are in control.",
+      "buttons": {
```

Comment:
nitpick: bumi and me discussed if this is better called `actions`. Might not always stay a button, etc.

Reply:
Very true, fixed!